_sass/_basecoat-tables.scss

```diff
@@ -22,3 +22,10 @@
     white-space: nowrap;
   }
 }
+
+.table-with-spacious-second-column td:nth-child(2) {
```

Comment:
Not a big deal, but since the styles are the same, you could chain the selector, like:

```scss
.table-with-spacious-first-column td:first-child,
.table-with-spacious-second-column td:nth-child(2) {
  // ...
}
```